src/help.h

```diff
@@ -130,12 +130,12 @@ struct commandHelp {
     15,
     "2.6.0" },
     { "BITFIELD",
-    "key [GET encoding offset] [SET encoding offset value] [INCRBY encoding offset increment] [OVERFLOW WRAP|SAT|FAIL]",
+    "key GET encoding offset|[OVERFLOW WRAP|SAT|FAIL] SET encoding offset value|INCRBY encoding offset increment [GET encoding offset|[OVERFLOW WRAP|SAT|FAIL] SET encoding offset value|INCRBY encoding offset increment ...]",
```

Comment:
looks like some parenthesis are needed, but that's out of the scope of this PR.
i suppose any element in a `oneof` or `block` with more than one element in it, should add `()`, but then we'll need redis-cli to handle that.
@jhelbaum FYI.

Reply:
If we decide to modify the syntax in general, we need to sync that with redis.io too.